docs/scripts/util/crawlAllLinksFromAllPages.ts

```diff
@@ -0,0 +1,41 @@
+import puppeteer from 'puppeteer';
+import { runArrayPromiseInOrder } from '.';
+
+export async function crawlAllLinksFromAllPages(allPages: string[]) {
+  type AllPagesPaths = Map<
+    string,
+    {
+      pageUrl: string;
+      links: { href: string; tagName: string; tagText: string }[];
+    }
+  >;
+  const allPagesPaths: AllPagesPaths = new Map();
+  await runArrayPromiseInOrder(allPages, checkSitemapPath);
+
+  return allPagesPaths;
```

Comment:
is `allPagesPaths` empty here?

Reply:
It's not empty, but it'd look better if I move it to the end of the function.